js/challenges/003_FindTwoNumbers/index.mjs

```diff
@@ -0,0 +1,22 @@
+/* 
+ * Returns an array of the indices of the two numbers in the input array that add to the target
+ * @param {array} arr Input array to search
+ * @param {int} target Target integer to search
+ * @returns {array} Two indices where numbers adding to target were found in given array
+ */
+export function findTwoNumbers (arr, target) {
+    var indices = [];
+    arr.forEach(findSum);
+    
+    function findSum(item, index) {
+        if (item < target) {
+            var remainder = (target - item);
+            var remainderIndex = arr.findIndex((x) => x === remainder);
```

Comment:
`findIndex` is a O(n) operation, which takes us back to the same time complexity of O(n^2), which is achievable with nested loops and no extra space.